docs/install/ha.md

```diff
@@ -0,0 +1,88 @@
+# High Availability
+
+This section describes how to install a high availability (HA) RKE2 cluster. An HA RKE2 cluster consists of:
+
+* A **fixed registration address** that is placed in front of server nodes to allow other nodes to register with the cluster
+* An odd number (three recommended) of **server nodes** that will serve the Kubernetes API and run other control plane services
+* Zero or more **agent nodes** that are designated to run your apps and services
+
+Agents register through the fixed registration address. However, when RKE2 launches the kubelet and it must connect to the Kubernetes api-server, it does so through the `rke2 agent` process, which acts as a client-side load balancer.
+
+Setting up an HA cluster requires the following steps:
+
+1. Configure a fixed registration address
+1. Launch the first server node
+1. Join additional server nodes
+1. Join agent nodes
+
+### 1. Configure the Fixed Registration Address
+
+Server nodes beyond the first one and all agent nodes need a URL to register against. This can be the IP or hostname of any of the server nodes, but in many cases those may change over time as nodes are created and destroyed. Therefore, you should have a stable endpoint in front of the server nodes.
+
+This endpoint can be set up using any number approaches, such as:
```

Comment:
any of the below approaches

Reply:
i think im going to leave it as-is, since these are just examples and other approaches not listed are valid.